manifests/moco-nodes.pp

```diff
@@ -342,6 +342,13 @@
     include toplevel::server::buildduty_tools
 }
 
+# mergeday
+
+node /^mergeday\d+\.srv\.releng\.(use1|usw2)\.mozilla\.com$/ {
+    $aspects = [ 'medium-security' ]
```

Comment:
Hm, I didn't dig into the specifics of what this encompasses, but is there any reason why this isn't `high-security`?

Reply:
I could change it, I guess. There's no secrets or other valuable information on this machine though - so I don't think it matters.

Reply:
A ssh-agent connection could probably be considered valuable?

Reply:
That's true. Eh, let's mark this high security then.